cli/lib/kontena/plugin_manager.rb

```diff
@@ -174,10 +174,9 @@ def load_plugins
                 $stderr.puts " [#{Kontena.pastel.red('error')}] Plugin #{Kontena.pastel.cyan(plugin_name)} (#{spec.version}) is not compatible with the current cli version."
                 $stderr.puts "         To update the plugin, run 'kontena plugin install #{plugin_name}'"
               end
-            rescue LoadError => ex
-              $stderr.puts " [#{Kontena.pastel.red('error')}] Failed to load plugin: #{spec.name}"
+            rescue Exception => ex
```

Comment:
> Why don't we just rescue LoadError?

That alone won't catch syntax errors or any runtime errors or whatever that may occur during plugin loading.

> That sounds like a bug to me.

If a plugin was loading forever that would be the only way to reach `kontena plugin uninstall`.

Of course we can only rescue LoadError and StandardError and let any other Exceptions go through the normal crashing mechanism.